### ISSUE TITLE
Use presets from polymer-build

### ISSUE THREAD

**Issue by bendavis78** [Collaborne/gulp-polymer-build-utils#6]
I used this expecting it to behave similarly to polymer-cli's `polymer build` command. The `polymer build` command creates directories for presets, such as `es5-bundled`, `es6-bundled`, and `es6-unbundled`. It would be nice if this plugin read presets from `polymer.json` during the build process in order to create separate builds.

**Comment by bendavis78** [comment]
Actually, now that I think of it, there should be a way to replicate each of the "Build Configuration" options from `polymer.json`. It doesn't look like this plugin does that?

**Comment by ronnyroeller** [comment]
> Actually, now that I think of it, there should be a way to replicate each of the "Build Configuration" options from polymer.json. It doesn't look like this plugin does that?

Indeed, it doesn't do this right but indeed it would make sense. I'd be happy to merge a PR if somebody wants to implement that.
